src/utilcode/debug.cpp

```diff
@@ -349,13 +349,6 @@ bool _DbgBreakCheck(
     STATIC_CONTRACT_FORBID_FAULT;
     STATIC_CONTRACT_DEBUG_ONLY;
 
-    RaiseExceptionOnAssert(rTestAndRaise);
-
-    if (DebugBreakOnAssert())
-    {
-        DebugBreak();
-    }
-
```

Comment:
Should we instead be fixing DebugBreakOnAssert instead of removing its usage? If it is completely broken on all platforms then I'll take that as evidence nobody uses it, but otherwise it seems moderately useful and perhaps there are other developers around the team still using it?